dev-tools/mage/dashboard.go

```diff
@@ -36,6 +36,8 @@ func ExportDashboard() error {
 		return fmt.Errorf("Dashboad ID must be specified")
 	}
 
+	kibanaURL := EnvOr("KIBANA", "")
```

Comment:
Should we call this `KIBANA_BASE_PATH` because to remove it in the kibana build you need to run `--no-base-path` flag.

Reply:
I prefer KIBANA_URL. That's what I used in the import target in the PR.

https://github.com/elastic/beats/pull/9842/files#diff-81a661bb38d6e9e450c732a07bdb4ac0R63

Reply:
I'm fine with either.

Reply:
me too, KIBANA_URL then? :-)

Reply:
Sounds good :)